docs/guides/user-interactions/readme.md

```diff
@@ -0,0 +1,182 @@
+---
+authors: Matt Richerson <matthew.richerson@hpe.com>
+categories: provisioning
+---
+
+# Rabbit User Interactions
+
+## Overview
+
+A user may include one or more Data Workflow directives in their job script to request Rabbit services. Directives take the form `#DW [command] [command args]`, and are passed from the workload manager to the Rabbit software for processing. The directives can be used to allocate Rabbit file systems, copy files, and run user containers on the Rabbit nodes.
```

Comment:
You use `[command]` here but then your next section is Directives. I think you should use the same terms here.